src/aws/region.rs

```diff
@@ -55,8 +55,8 @@ mod tests {
 
     #[test]
     fn optional() {
-        assert!(toml::from_str::<RegionOrEndpoint>(indoc! {r#"
-        "#})
+        assert!(toml::from_str::<RegionOrEndpoint>(indoc! {r"
```

Comment:
```suggestion
        assert!(toml::from_str::<RegionOrEndpoint>(indoc! {"
```

Reply:
apply all of this in the rest of the changes please too please. It seems that the raw strings were also not necessary